### PR TITLE
Clean up formatting in tournament articles

### PR DESCRIPTION
**Pull request by TPGPL**
part 2 of clean up in tournament articles

also its probably better to review commit-by-commit than by diff

### CODE REVIEW

**Review by Walavouchey — COMMENTED** [review]
seems otherwise alright

**Comment by TicClick** [comment]
today

File: wiki/Tournaments/MWC/2015/en.md [hunk 1]
```diff
@@ -446,9 +446,9 @@ Sunday, 23 August 2015:
 3. Each captain can ban two beatmaps to be selected from the pool. These beatmaps are not allowed to be picked by any team in the entire match.
    - Both vetos must always be used.
 4. Beatmap selection will alternate between each captain selecting a beatmap out of the map pool.
-5. Each captain must use "!roll" once in \#multiplayer.
-   - The winner of the !roll starts picking the first beatmap of the match.
-   - The loser of the !roll starts banning two beatmaps.
+5. Each captain must use `!roll` once in \#multiplayer.
```
Comment (TicClick) [inline wiki/Tournaments/MWC/2015/en.md:449]:
```suggestion
5. Each captain must use `!roll` once in #multiplayer.
```

File: wiki/Tournaments/MWC/2016_4K/en.md [hunk 2]
```diff
@@ -460,9 +460,9 @@ Sunday, 7 August 2016:
 2. Players are free to select up to two warm-up maps. Using beatmaps with questionable content is prohibited.
 3. Each captain must ban one beatmap to be selected from the pool. These beatmaps are not allowed to be picked by any team in the entire match.
 4. Beatmap selection will alternate between each captain selecting a beatmap out of the map pool.
-5. Each captain must use "!roll" once in \#multiplayer.
-   - The winner of the !roll starts picking the first beatmap of the match.
-   - The loser of the !roll starts banning.
+5. Each captain must use `!roll` once in \#multiplayer.
```
Comment (TicClick) [inline wiki/Tournaments/MWC/2016_4K/en.md:463]:
```suggestion
5. Each captain must use `!roll` once in #multiplayer.
```

File: wiki/Tournaments/MWC/2016_7K/en.md [hunk 3]
```diff
@@ -435,9 +435,9 @@ Sunday, 10 January 2016:
 3. Each captain can ban two beatmaps to be selected from the pool. These beatmaps are not allowed to be picked by any team in the entire match.
    - Both vetos must always be used.
 4. Beatmap selection will alternate between each captain selecting a beatmap out of the map pool.
-5. Each captain must use "!roll" once in \#multiplayer.
-   - The winner of the !roll starts picking the first beatmap of the match.
-   - The loser of the !roll starts banning two beatmaps.
+5. Each captain must use `!roll` once in \#multiplayer.
```
Comment (TicClick) [inline wiki/Tournaments/MWC/2016_7K/en.md:438]:
```suggestion
5. Each captain must use `!roll` once in #multiplayer.
```

File: wiki/Tournaments/OWC/2013/en.md [hunk 4]
```diff
@@ -463,7 +463,7 @@ Sunday, 10 November 2013:
    - The team mentioned first in the room name must be the blue team, the team mentioned second in the room name must be the red team.
 2. Referee will join the room externally. Referee will spectate the match via multi-spectator client.
 3. Players are free to select a warm-up map.
-4. Map selection will alternate between each captain selecting a map out of the map pool. Each captain must use "!roll" once in \#multiplayer to determine which team selects first.
+4. Map selection will alternate between each captain selecting a map out of the map pool. Each captain must use `!roll` once in \#multiplayer to determine which team selects first.
```
Comment (TicClick) [inline wiki/Tournaments/OWC/2013/en.md:466]:
```suggestion
4. Map selection will alternate between each captain selecting a map out of the map pool. Each captain must use `!roll` once in #multiplayer to determine which team selects first.
```

File: wiki/Tournaments/OWC/2013/it.md [hunk 5]
```diff
@@ -105,7 +101,7 @@ L'osu! World Cup 2013 è un torneo creato dallo [Staff Tornei](https://osu.ppy.s
    3. The team mentioned first in the room name must be the blue team, the team mentioned second in the room name must be the red team.
 2. Referee will join the room externally. Referee will spectate the match via multi-spectator client.
 3. Players are free to select a warm-up map.
-4. Map selection will alternate between each captain selecting a map out of the map pool. Each captain must use "!roll" once in \#multiplayer to determine which team selects first.
+4. Map selection will alternate between each captain selecting a map out of the map pool. Each captain must use `!roll` once in \#multiplayer to determine which team selects first.
```
Comment (TicClick) [inline wiki/Tournaments/OWC/2013/it.md:104]:
```suggestion
4. Map selection will alternate between each captain selecting a map out of the map pool. Each captain must use `!roll` once in #multiplayer to determine which team selects first.
```

File: wiki/Tournaments/OWC/2013/pt.md [hunk 6]
```diff
@@ -347,7 +337,7 @@ A Copa do Mundo do osu! 2013 (em inglês, osu! World Cup 2013), também conhecid
    3. A equipe mencionada primeiramente no nome da sala será a equipe azul, a equipe mencionada posteriormente será a equipe vermelha.
 2. O árbitro entrará na sala externamente. O árbitro irá assistir a partida por meio de um client especial para multi-spectating.
 3. Os jogadores estarão livres para escolher um beatmap de aquecimento.
-4. A seleção de mapas será alternada entre cada capitão escolhendo um beatmap por vez. Os capitães deverão usar o comando "!roll" no \#multiplayer para determinar qual a equipe que escolherá primeiro.
+4. A seleção de mapas será alternada entre cada capitão escolhendo um beatmap por vez. Os capitães deverão usar o comando `!roll` no \#multiplayer para determinar qual a equipe que escolherá primeiro.
```
Comment (TicClick) [inline wiki/Tournaments/OWC/2013/pt.md:340]:
```suggestion
4. A seleção de mapas será alternada entre cada capitão escolhendo um beatmap por vez. Os capitães deverão usar o comando `!roll` no #multiplayer para determinar qual a equipe que escolherá primeiro.
```

File: wiki/Tournaments/OWC/2013/ru.md [hunk 7]
```diff
@@ -169,7 +165,7 @@ outdated: true
    3. Первая команда, упомянутая в шаблоне, должна занять синюю сторону, вторая — красную.
 2. Судья будет наблюдать за игрой с помощью специальной версии игрового клиента.
 3. Разминочная битмапа выбирается по договоренности между игроками команд.
-4. Битмапы из пула будут выбираться капитанами команда по очереди. Капитан, выбирающий первым, будет выбран жеребьевкой, которая будет проводиться с помоью команды !roll. Её нужно написать в канал \#multiplayer один раз. Тот, кому выпадет наибольшее число, и будет выбирать первым.
+4. Битмапы из пула будут выбираться капитанами команда по очереди. Капитан, выбирающий первым, будет выбран жеребьевкой, которая будет проводиться с помоью команды `!roll`. Её нужно написать в канал \#multiplayer один раз. Тот, кому выпадет наибольшее число, и будет выбирать первым.
```
Comment (TicClick) [inline wiki/Tournaments/OWC/2013/ru.md:168]:
```suggestion
4. Битмапы из пула будут выбираться капитанами команда по очереди. Капитан, выбирающий первым, будет выбран жеребьевкой, которая будет проводиться с помоью команды `!roll`. Её нужно написать в канал #multiplayer один раз. Тот, кому выпадет наибольшее число, и будет выбирать первым.
```

File: wiki/Tournaments/OWC/2014/en.md [hunk 8]
```diff
@@ -487,8 +487,8 @@ Sunday, 9 November 2014:
 5. Players are free to select up to two warm-up beatmaps. Using beatmaps with questionable content is prohibited.
 6. Each captain can ban two beatmaps to be selected from the pool. These beatmaps are not allowed to be picked by any team in the entire match.
 7. Both vetos must always be used.
-8. Both captains must use "!roll" to determine, who starts banning two beatmaps. The other captain can not ban the same beatmaps and must pick another beatmap in case of an overlap.
-9. Beatmap selection will alternate between each captain selecting a beatmap out of the map pool. Each captain must use "!roll" once in \#multiplayer to determine which team selects first.
+8. Both captains must use `!roll` to determine, who starts banning two beatmaps. The other captain can not ban the same beatmaps and must pick another beatmap in case of an overlap.
+9. Beatmap selection will alternate between each captain selecting a beatmap out of the map pool. Each captain must use `!roll` once in \#multiplayer to determine which team selects first.
```
Comment (TicClick) [inline wiki/Tournaments/OWC/2014/en.md:491]:
```suggestion
9. Beatmap selection will alternate between each captain selecting a beatmap out of the map pool. Each captain must use `!roll` once in #multiplayer to determine which team selects first.
```

File: wiki/Tournaments/OWC/2014/ko.md [hunk 9]
```diff
@@ -133,8 +129,8 @@ outdated: true
 2. Players are free to select up to two warm-up beatmaps. Using beatmaps with questionable content is prohibited.
 3. Each captain can ban two beatmaps to be selected from the pool. These beatmaps are not allowed to be picked by any team in the entire match.
    1. Both vetos must always be used.
-   2. Both captains must use "!roll" to determine, who starts banning two beatmaps. The other captain can not ban the same beatmaps and must pick another beatmap in case of an overlap.
-4. Beatmap selection will alternate between each captain selecting a beatmap out of the map pool. Each captain must use "!roll" once in \#multiplayer to determine which team selects first.
+   2. Both captains must use `!roll` to determine, who starts banning two beatmaps. The other captain can not ban the same beatmaps and must pick another beatmap in case of an overlap.
+4. Beatmap selection will alternate between each captain selecting a beatmap out of the map pool. Each captain must use `!roll` once in \#multiplayer to determine which team selects first.
```
Comment (TicClick) [inline wiki/Tournaments/OWC/2014/ko.md:133]:
```suggestion
4. Beatmap selection will alternate between each captain selecting a beatmap out of the map pool. Each captain must use `!roll` once in #multiplayer to determine which team selects first.
```

File: wiki/Tournaments/OWC/2014/pt.md [hunk 10]
```diff
@@ -316,7 +304,7 @@ A Copa do Mundo do osu! 2014 (em inglês, osu! World Cup 2014), também conhecid
    3. A equipe mencionada primeiramente no nome da sala será a equipe azul, a equipe mencionada posteriormente será a equipe vermelha.
 2. O árbitro entrará na sala externamente. O árbitro irá assistir a partida por meio de um client especial para multi-spectating.
 3. Os jogadores estarão livres para escolher um beatmap de aquecimento.
-4. A seleção de mapas será alternada entre cada capitão escolhendo um beatmap por vez. Os capitães deverão usar o comando "!roll" no \#multiplayer para determinar qual a equipe que escolherá primeiro.
+4. A seleção de mapas será alternada entre cada capitão escolhendo um beatmap por vez. Os capitães deverão usar o comando `!roll` no \#multiplayer para determinar qual a equipe que escolherá primeiro.
```
Comment (TicClick) [inline wiki/Tournaments/OWC/2014/pt.md:307]:
```suggestion
4. A seleção de mapas será alternada entre cada capitão escolhendo um beatmap por vez. Os capitães deverão usar o comando `!roll` no #multiplayer para determinar qual a equipe que escolherá primeiro.
```

File: wiki/Tournaments/OWC/2015/de.md [hunk 11]
```diff
@@ -232,9 +226,9 @@ Wir versuchen ein Bargeld-Preispool von mindestens 5000$ für diese WM zu erreic
 3. Jeder Kapitän kann 2 Beatmaps aus dem Pool wählen, die nicht gespielt werden sollen. Dies gilt für beide Teams und die komplette Partie hindurch.
    1. Beide Vetos müssen benutzt werden.
 4. Die Beatmaps werden aus dem jeweiligen Pool abwechselnd von den beiden Kapitänen ausgewählt.
-5. Beide Kapitäne müssen "!roll" einmal im \#multiplayer benutzen.
-   1. Der Gewinner beim !roll wählt die zuerst spielende Map der Partie aus.
-   2. Der Verlierer beim !roll beginnt mit dem Bannen von 2 Beatmaps.
+5. Beide Kapitäne müssen `!roll` einmal im \#multiplayer benutzen.
```
Comment (TicClick) [inline wiki/Tournaments/OWC/2015/de.md:229]:
```suggestion
5. Beide Kapitäne müssen `!roll` einmal im #multiplayer benutzen.
```

File: wiki/Tournaments/OWC/2015/en.md [hunk 12]
```diff
@@ -494,9 +494,9 @@ Sunday, 8 November 2015:
 3. Each captain can ban two beatmaps to be selected from the pool. These beatmaps are not allowed to be picked by any team in the entire match.
    - Both vetos must always be used.
 4. Beatmap selection will alternate between each captain selecting a beatmap out of the map pool.
-5. Each captain must use "!roll" once in \#multiplayer.
-   - The winner of the !roll starts picking the first beatmap of the match.
-   - The loser of the !roll starts banning two beatmaps.
+5. Each captain must use `!roll` once in \#multiplayer.
```
Comment (TicClick) [inline wiki/Tournaments/OWC/2015/en.md:497]:
```suggestion
5. Each captain must use `!roll` once in #multiplayer.
```

File: wiki/Tournaments/OWC/2015/ja.md [hunk 13]
```diff
@@ -349,7 +349,7 @@ osu! World Cup 2015は、様々なコミュニティのメンバーによって
 3. それぞれのキャプテンはビートマップ群から2つビートマップを使用禁止にすることができます。ここで選ばれたビートマップは試合を通してどちらのチームも使用してはいけません。
    1. このビートマップ拒否権は必ずどちらも使用してください。
 4. ビートマップの選択は、各チームキャプテンが交互に行います。
-5. それぞれのキャプテンは\#multiplayerで"!roll"を使ってください。
+5. それぞれのキャプテンは\#multiplayerで`!roll`を使ってください。
```
Comment (TicClick) [inline wiki/Tournaments/OWC/2015/ja.md:352]:
```suggestion
5. それぞれのキャプテンは#multiplayerで`!roll`を使ってください。
```

File: wiki/Tournaments/OWC/2015/ko.md [hunk 14]
```diff
@@ -409,9 +397,9 @@ outdated: true
 3. Each captain can ban two beatmaps to be selected from the pool. These beatmaps are not allowed to be picked by any team in the entire match.
    1. Both vetos must always be used.
 4. Beatmap selection will alternate between each captain selecting a beatmap out of the map pool.
-5. Each captain must use "!roll" once in \#multiplayer.
-   1. The winner of the !roll starts picking the first beatmap of the match.
-   2. The loser of the !roll starts banning two beatmaps.
+5. Each captain must use `!roll` once in \#multiplayer.
```
Comment (TicClick) [inline wiki/Tournaments/OWC/2015/ko.md:400]:
```suggestion
5. Each captain must use `!roll` once in #multiplayer.
```

File: wiki/Tournaments/OWC/2015/zh.md [hunk 15]
```diff
@@ -541,9 +527,9 @@ outdated: true
 3. Each captain can ban two beatmaps to be selected from the pool. These beatmaps are not allowed to be picked by any team in the entire match.
    1. Both vetos must always be used.
 4. Beatmap selection will alternate between each captain selecting a beatmap out of the map pool.
-5. Each captain must use "!roll" once in \#multiplayer.
-   1. The winner of the !roll starts picking the first beatmap of the match.
-   2. The loser of the !roll starts banning two beatmaps.
+5. Each captain must use `!roll` once in \#multiplayer.
```
Comment (TicClick) [inline wiki/Tournaments/OWC/2015/zh.md:530]:
```suggestion
5. Each captain must use `!roll` once in #multiplayer.
```

File: wiki/Tournaments/SOL/2019/en.md [hunk 16]
```diff
@@ -592,7 +586,7 @@ Sunday, 08 December 2019:
 - Players are free to select one beatmap per player for warmup. Beatmaps with questionable or explicit content is prohibited. All the beatmaps must be osu! beatmaps specifically, and must be submitted so each player can download it.
 - Each player can ban a beatmap from being selected in the match. These beatmaps won’t be selected by any player throughout the match.
 - Beatmap selection will alternate between each player selecting a beatmap from the mappool.
-- Each player must use "!roll" once in #multiplayer. -The loser will choose the first ban of the match. The winner will choose the second ban and the first pick of the match.
+- Each player must use `!roll` once in #multiplayer. -The loser will choose the first ban of the match. The winner will choose the second ban and the first pick of the match.
```
Comment (TicClick) [inline wiki/Tournaments/SOL/2019/en.md:589]:
```suggestion
- Each player must use `!roll` once in #multiplayer.
- The loser will choose the first ban of the match. The winner will choose the second ban and the first pick of the match.
```

File: wiki/Tournaments/TWC/2014/zh.md [hunk 17]
```diff
@@ -117,7 +113,7 @@ outdated: true
    2. Room settings are osu!, Team-Vs., Win Condition: 'Score'. Room name must be "Taiko World Cup 2014: TeamBlue vs TeamRed"
    3. The team mentioned first in the room name must be the blue team, the team mentioned second in the room name must be the red team.
 2. Players are free to select up to two warm-up maps.
-3. Map selection will alternate between each captain selecting a map out of the map pool. Each captain must use "!roll" once in \#multiplayer to determine which team selects first.
+3. Map selection will alternate between each captain selecting a map out of the map pool. Each captain must use `!roll` once in \#multiplayer to determine which team selects first.
```
Comment (TicClick) [inline wiki/Tournaments/TWC/2014/zh.md:116]:
```suggestion
3. Map selection will alternate between each captain selecting a map out of the map pool. Each captain must use `!roll` once in #multiplayer to determine which team selects first.
```

File: wiki/Tournaments/TWC/2015/de.md [hunk 18]
```diff
@@ -128,9 +124,9 @@ Das **osu! Taiko World Cup 2015** (***TWC 2015***) ist ein länderbezogenes Taik
 3. Jeder Kapitän kann 2 Beatmaps aus dem Pool wählen, die nicht gespielt werden sollen. Dies gilt für beide Teams und die komplette Partie hindurch.
    1. Beide Vetos müssen benutzt werden.
 4. Die Beatmaps werden aus dem jeweiligen Pool abwechselnd von den beiden Kapitänen ausgewählt.
-5. Beide Kapitäne müssen einmal "!roll" ins \#multiplayer-Channel eingeben.
-   1. Der Gewinner beim !roll wählt die zuerst spielende Map der Partie aus.
-   2. Der Verlierer beim !roll beginnt mit dem Bannen von 2 Beatmaps.
+5. Beide Kapitäne müssen einmal `!roll` ins \#multiplayer-Channel eingeben.
```
Comment (TicClick) [inline wiki/Tournaments/TWC/2015/de.md:127]:
```suggestion
5. Beide Kapitäne müssen einmal `!roll` ins #multiplayer-Channel eingeben.
```

File: wiki/Tournaments/TWC/2015/es.md [hunk 19]
```diff
@@ -137,9 +133,9 @@ El país campeón del torneo anterior es ![][flag_JP] **Japón**.
 3. Cada capitán podrá vetar 2 beatmaps para ser seleccionado del sorteo, esos beatmaps no serán elegibles para ningún otro equipo en ninguna partida.
    1. Ambos vetos podrán ser utilizados cuando se requiera.
 4. La selección de beatmaps se alternarará entre cada capitán seleccionado un beatmap fuera del sorteo de mapas.
-5. Cada capitan podrá usar el comando !roll en \#multiplayer.
-   1. El ganador del !roll podrá elegir primero el beatmap de la partida.
-   2. El perdedor del !roll empezará por vetar 1 o 2 beatmaps.
+5. Cada capitan podrá usar el comando `!roll` en \#multiplayer.
```
Comment (TicClick) [inline wiki/Tournaments/TWC/2015/es.md:136]:
```suggestion
5. Cada capitan podrá usar el comando `!roll` en #multiplayer.
```

File: wiki/Tournaments/TWC/2015/ja.md [hunk 20]
```diff
@@ -173,9 +169,9 @@ outdated: true
 3. それぞれのキャプテンはビートマップ群から2つビートマップを使用禁止にすることができます。ここで選ばれたビートマップは試合を通してどちらのチームも使用してはいけません。
    1. このビートマップ拒否権は必ずどちらも使用してください。
 4. ビートマップ選択はそれぞれのキャプテンが交互に行います。
-5. それぞれのキャプテンは\#multiplayerで"!roll"を使ってください。
-   1. !rollの勝者は先に試合のビートマップを選んでください。
-   2. !rollの敗者は先に2つビートマップを使用禁止にしてください。
+5. それぞれのキャプテンは\#multiplayerで`!roll`を使ってください。
```
Comment (TicClick) [inline wiki/Tournaments/TWC/2015/ja.md:172]:
```suggestion
5. それぞれのキャプテンは#multiplayerで`!roll`を使ってください。
```

File: wiki/Tournaments/TWC/2015/ru.md [hunk 21]
```diff
@@ -112,9 +108,9 @@ outdated: true
 4. Первые две команды в каждой группе проходят в следующий этап.
    1. В зависимости от количества команд, возможны изменения.
 
-5. Дальнейшие этапы проводятся по [системе с двойным утешением](https://ru.wikipedia.com/Олимпийская_система_с_выбыванием_после_двух_поражений): выигравшая команда переходит в следующий этап, проигравшая попадает в сетку проигравших.
+5. Дальнейшие этапы проводятся по [системе с двойным утешением](https://ru.wikipedia.com/Олимпийская_система_с_выбыванием_после_двух_поражений "Wikipedia"): выигравшая команда переходит в следующий этап, проигравшая попадает в сетку проигравших.
```
Comment (TicClick) [inline wiki/Tournaments/TWC/2015/ru.md:111]: this is a very odd link, I suspect there may be more
```suggestion
5. Дальнейшие этапы проводятся по [системе с двойным утешением](https://ru.wikipedia.org/wiki/Олимпийская_система_с_выбыванием_после_двух_поражений "Wikipedia"): выигравшая команда переходит в следующий этап, проигравшая попадает в сетку проигравших.
```

**Review by TicClick — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/TWC/2015/ru.md (hunk 21 above)
Comment (TPGPL) [reply wiki/Tournaments/TWC/2015/ru.md]: looks like this was the only broken wikipedia link in the `/Tournaments/` directory

**Review by TicClick — CHANGES_REQUESTED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**